site/content/integrate/apps/_index.md

```diff
@@ -1,34 +1,51 @@
 ---
 title: "Apps (Developers Preview)"
-heading: "Overview"
+heading: "Introduction"
 description: "Apps are lightweight interactive add-ons to Mattermost."
 section: "integrate"
 weight: 90
 ---
 
-Apps are lighweight interactive add-ons to Mattermost. Apps can:
+Apps are lightweight, interactive add-ons to Mattermost which use serverless hosting to run without any dedicated infrastructure. Apps can:
 
 - Display interactive, dynamic Modal forms and Message Actions.
-- Attach themselves to locations in the Mattermost UI (e.g. channel bar buttons, post menu, channel menu, commands), and can add their custom /commands with full Autocomplete.
+- Be written in several different languages.
+- Attach themselves to locations in the Mattermost user interface (e.g. channel bar buttons, post menu, channel menu, commands), and can add their custom `/` commands with full Autocomplete.
 - Receive webhooks from Mattermost, and third-parties, and use the Mattermost REST APIs to post messages, etc.
-- Be hosted externally (HTTP), on Mattermost Cloud (AWS Lambda), and soon in self-managed as well as customers' own AWS environments.
-- Be developed in any language*.
+- Be hosted externally (HTTP) and on Mattermost Cloud (AWS Lambda)*.
+- Work on both Mobile and Desktop apps so developers can focus on the functionality of their apps.
+- Be deployed using our Mattermost serverless hosting infrastructure keeping data secure and supporting scalability.
 
 Apps are available as a Developers Preview and we're looking for your feedback! Share constructive feedback in the [Mattermost Apps channel](https://community.mattermost.com/core/channels/mattermost-apps) on our Mattermost community instance.
 
+* Apps will soon be available for self-managed deployments as well as customers' own AWS environments.
+
 Read the [quick start guide]({{< ref  "quick-start-go" >}}) to learn how to write your first app.
 
-# Contents
+## Mattermost serverless app hosting
+
+Mattermost serverless hosting allows for easy app installation from the App Marketplace by a System Admin and uses AWS Lambda serverless technology instead of relying on a physical server. Developers who create apps using a serverless development approach can easily deploy apps securely, efficiently, and at scale in the Mattermost Cloud.
```

Comment:
3/5 that hosting is not such an important topic that we need to talk about it upfront. It's a topic for developers, not for admins.

Reply:
I see this as a combination admin/dev guide. Both parties need context but the content can be trimmed down a lot.

Reply:
But re-reading it, you're correct and I have removed the content. :)